walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,52 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.Collection; 
+import java.util.Set;
+import java.util.HashSet;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> available_times = new ArrayList<TimeRange>();
+    if (request.getDuration() > TimeRange.WHOLE_DAY.duration()) { // not possible for duration to exceed whole day
+        return available_times;
+    }
+    ArrayList<TimeRange> blocked_times = new ArrayList<TimeRange>();
```

Comment:
Nit here and below: Java style uses lower camel case for local variables: `blockedTimes`